dags/pubmed.py

```diff
@@ -22,15 +22,22 @@
     command='make start pubmed 1900-01-01 2100-01-01'
 )
 
-processor_task = helpers.create_processor_task(
-    name='pubmed',
+unregistred_trials_task = helpers.create_processor_task(
+    name='pubmed_unregistered_trials',
     dag=dag
 )
 
-pubmed_linker_task = helpers.create_processor_task(
-    name='pubmed_linker',
+trials_remover_task = helpers.create_processor_task(
+    name='trial_remover',
     dag=dag
 )
 
-processor_task.set_upstream(collector_task)
-pubmed_linker_task.set_upstream(processor_task)
+publications_task = helpers.create_processor_task(
+    name='pubmed_publications',
+    dag=dag
+)
+
+
+unregistred_trials_task.set_upstream(collector_task)
+trials_remover_task.set_upstream(unregistred_trials_task)
+publications_task.set_upstream(trials_remover_task)
```

Comment:
It seems to me that order and dependency here are the same thing. For me, dependency here is "Would the results of this task influence the other tasks?". This seems to be the case here, so all's fine with the code.